services/app-web/src/components/ExpandableText/ExpandableText.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useLayoutEffect, useState } from 'react'
+import { Button } from '@trussworks/react-uswds'
+import styles from './ExpandableText.module.scss'
+
+export type ExpandableTextProps = {
+    clamp: React.ReactElement | string // you can pass in an element with nested <span> or simple string
+    clampedLines?: number
+}
+
+export const ExpandableText = ({
+    clamp,
```

Comment:
Since this component wraps other components, could we use children here instead of passing react elements in as props?

Reply:
We could, this was where I'm unsure if we want to pass elements at all ( only strings ). I got stuck on how to add bold text to the `Reason for unlock:` and `Changes made:` text with out passing in elements. Being able to pass any element(s) causes unexpected/unintentional clamping results.

Reply:
fwiw I did look into only using a string and I agree we may have to pass elements to match the current design, like you noticed @JasonLin0991 .  I also  added a comment about only using <span>s in my commit. You are right that it could have unintentional results if complex nested HTML is passed, but I think having a comment is the best we can do while still making this component flexible

How do you feel the proposal to render children instead of using a prop @JasonLin0991 ?

Reply:
`How do you feel the proposal to render children instead of using a prop @JasonLin0991 ?`

Works for me!